examples/blinky/main.c

```diff
@@ -41,7 +42,7 @@ static void delay(void)
          * compilers would detect that the loop is only wasting CPU cycles and
          * optimize it out - but here the wasting of CPU cycles is desired.
          */
-        for (volatile uint32_t i = 0; i < CLOCK_CORECLOCK / 20; i++) { }
+        for (volatile uint32_t i = 0; i < coreclk() / 20; i++) { }
```

Comment:
I assume it is guaranteed that `coreclk()` always is either a read of a non-volatile global or a const; but somehow when i read this i assume there might be a function call and a beginner might be inspired by this. ->
```suggestion
        uint32_t runs = coreclk() / 20;
        for (volatile uint32_t i = 0; i < runs; i++) { }
```